### PR TITLE
Layout stories pages

### PR DESCRIPTION
**Pull request by ericyoungberg**
Resolves _no specific issue_.

# Why is this change necessary?
Add more layout to the stories section.

### CODE REVIEW

File: app/assets/stylesheets/global/_extends.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+._clear {
```
Comment (mzagaja) [inline app/assets/stylesheets/global/_extends.scss]: Why does this start with an underscore?

File: app/assets/stylesheets/refinery/stories.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
+.StoriesPage {
```
Comment (mzagaja) [inline app/assets/stylesheets/refinery/stories.scss]: I like this refactor generally but am not sure it should fit inside "refinery" structure in the app because the stories feature was built outside the structure of refinery (though I did suggest that at some point we might want to refactor it as a refinery extension).

File: app/views/refinery/pages/home.html.erb [hunk 3]
```diff
@@ -39,7 +39,6 @@
           <% end %>
         </div>
         <div class="content-column-two">
-
```
Comment (mzagaja) [inline app/views/refinery/pages/home.html.erb]: While I agree with this minor update I generally prefer to keep things where we prettify to a separate PR to avoid misdirection when perusing git history.

File: app/views/stories/_form.html.erb [hunk 4]
```diff
@@ -1,36 +1,50 @@
...
+      <%= form.label :video, t('stories.upload_or_take_video') %>
+    </div>
+
+    <div class="form-break">or</div>
+
+    <div class="field audio-response" id="audio-recording">
+      <h4>Audio Recording</h4>
+
+      <div class="audio-buttons">
+        <button type="button" id="recordButton" class="button">Start Recording</button>
```
Comment (mzagaja) [inline app/views/stories/_form.html.erb]: I know I didn't do this but we should probably turn these text blobs into i18n thingies (`<%= t('start_recording') %>`). You can either do that here or I'll just make another PR.

File: app/assets/stylesheets/global/_extends.scss (hunk 1 above)
Comment (ericyoungberg) [reply app/assets/stylesheets/global/_extends.scss]: Most languages use an underscore to mark privacy. If the value is written with an underscore, it shouldn't be used outside of an implementer. 

In this case, the implementer is Sass' `@extend` macro; the `._clear` class should never be used directly, such as `<div class="row _clear"></div>`.

File: app/assets/stylesheets/refinery/stories.scss (hunk 2 above)
Comment (ericyoungberg) [reply app/assets/stylesheets/refinery/stories.scss]: :ok_hand: 

I was just trying to follow whatever the current directory structure looked like. I'll change where this file lives.

File: app/views/stories/_form.html.erb (hunk 4 above)
Comment (ericyoungberg) [reply app/views/stories/_form.html.erb]: Yeah, let's separate that PR.
